cluster-sharding/src/main/scala/com/evolutiongaming/akkaeffect/cluster/sharding/ClusterShardingLocal.scala

```diff
@@ -133,9 +133,16 @@ object ClusterShardingLocal {
                 Map((self, shardIds))
               }
 
+              var rebalancing = false
+              var stashed     = Vector.empty[ShardRegion.Msg]
+
+              object Unstash
+
               def receive: Receive = {
 
                 case RegionMsg.Rebalance =>
+                  rebalancing = true
+                  context.system.scheduler.scheduleOnce(1.second, self, Unstash)
```

Comment:
is this for some local testing only? why `1.second` and not something else? why not `Unstash` in `onComplete` of rebalance?

Reply:
why we need explicit `Unstash`? why we cannot do whole unstashing in `onComplete` callback?

Reply:
> is this for some local testing only?

yes, `ClusterShardingLocal` is test impl of `ClusterSharding` used purely in unit-tests.

> why 1.second and not something else?

I see it as reasonable time for election of new region for the shard. Works well in tests now, later if needed will make configurable.

> why not `Unstash` in `onComplete` of rebalance?

`allocationStrategy` is just part of rebalance and probably its test impl with instant response thus async 
```
                      context
                        .actorSelection("*/*")
                        .tell(handOffStopMessage, self)
```
will not complete before as its guaranteed by Akka.

Reply:
>used purely in unit-tests.

why it is not in `test` scope? or maybe it should be in dedicated `*-testkit` module instead?

Reply:
this Q not for me )

as for now I prefer to keep it as is because my goal not in refactoring rather make it working for proper rebalance tests